migrationmanager/migrators/metrics/migrations/000003_ts_v4.up.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE IF NOT EXISTS signoz_metrics.time_series_v4 ON CLUSTER {{.SIGNOZ_CLUSTER}} (
+    env LowCardinality(String) DEFAULT 'default',
+    temporality LowCardinality(String) DEFAULT 'Unspecified',
+    metric_name LowCardinality(String),
+    description LowCardinality(String) DEFAULT '' CODEC(ZSTD(1)),
+    unit LowCardinality(String) DEFAULT '' CODEC(ZSTD(1)),
+    type LowCardinality(String) DEFAULT '' CODEC(ZSTD(1)),
+    is_monotonic Bool DEFAULT false CODEC(ZSTD(1)),
+    fingerprint UInt64 CODEC(Delta, ZSTD),
+    unix_milli Int64 CODEC(Delta, ZSTD),
+    labels String CODEC(ZSTD(5)),
+    INDEX idx_labels labels TYPE ngrambf_v1(4, 1024, 3, 0) GRANULARITY 1
+)
+ENGINE = ReplacingMergeTree
+        PARTITION BY toDate(unix_milli / 1000)
+        ORDER BY (env, temporality, metric_name, fingerprint, unix_milli)
+        TTL toDateTime(unix_milli/1000) + INTERVAL 2592000 SECOND DELETE;
```

Comment:
`SETTINGS ttl_only_drop_parts=1;` not needed?

Reply:
This table is already better off wrt TTL as the rows would have hourly precision and the table size should be in a reasonable range, unlike the other tables. I will add `ttl_only_drop_parts`.